modules/nw-operator-cr.adoc

```diff
@@ -120,10 +120,11 @@ configuration.
 are `Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is
 `NetworkPolicy`.
 
-<4> The maximum transmission unit (MTU) for the VXLAN overlay network. This
-value is normally configured automatically, but if the nodes in your cluster do
-not all use the same MTU, then you must set this explicitly to 50 less than the
-smallest node MTU value.
+<4> The maximum transmission unit (MTU) for the VXLAN overlay network. This is detected automatically based on the MTU of the primary network interface. You do not normally need to override the detected MTU.
++
+If the auto-detected value is not what you expected it to be, confirm that the MTU that is set on your nodes is correct. You cannot use this value to change the MTU value on the underlying nodes.
```

Comment:
```suggestion
If the auto-detected value is not what you expected it to be, confirm that the MTU on the primary network interface on your nodes is correct. You cannot use this option to change the MTU value of the primary network interface on the nodes.
```

ok that's a little bit redundant, but especially in the first sentence I thought it was important to clarify that we mean "check that the network interface has the correct MTU", not "check that that the VXLAN tunnel has the correct MTU"